### PR TITLE
Set up pre-commit: generate WES API

### PR DESCRIPTION
**Pull request by scvannost**

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    )
+    if output:
+        print(output)
+    if err:
+        os.exit(err)
+
+    output, err = system(
+        "git",
+        "add",
+        os.path.join("cidc_ngs_pipeline_api", "wes", "wes_output_API.json"),
```
Comment (curlup) [inline .githooks/pre-commit:37]: same thing twice?

**Comment by curlup** [comment]
would be nice to have instructions on installing pre-commit in the Readme

File: .githooks/pre-commit (hunk 1 above)
Comment (scvannost) [reply .githooks/pre-commit]: fix in next commit

**Review by roshni-b — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -61,3 +61,17 @@ Repository structure:
...
+Install necessary dependencies.
+
+```bash
+pip install -r requirements.dev.txt
+```
+
+Install and configure pre-commit hooks.
+
+```bash
+pre-commit install
```
Comment (curlup) [inline README.md:76]: shouldn't pre-commit also be a dependency? I don't see it there

**Pull request merged**